README.md

```diff
@@ -1,16 +1,19 @@
 Developer guide to Tessel 2 Docs Gitbook
```

Comment:
Note to you/also self: once we get this on gitbooks.io, we can use their simple editor - which doesn't require you to know how to set up a dev environment